### PR TITLE
fix: API slots endpoint return format

### LINKED ISSUE #13376: [CAL-3008] API: /slots returns inconsistently if you omit timeZone
An enterprise client reported this issue.

Made a local test to reproduce:

Will publish a fix soon.

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-3008](https://linear.app/calcom/issue/CAL-3008/api-slots-returns-inconsistently-if-you-omit-timezone)</sub>

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

Fixes #13376

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

- yarn test apps/api/pages/api/slots/_get.test.ts

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: apps/api/pages/api/slots/_get.test.ts
Comment (zomars) [inline apps/api/pages/api/slots/_get.test.ts:1]: Added unit test to prevent regressions

File: apps/api/pages/api/slots/_get.ts [hunk 1]
```diff
@@ -38,7 +38,7 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
           },
           {}
         )
-      : availableSlots;
+      : availableSlots.slots;
```
Comment (zomars) [inline apps/api/pages/api/slots/_get.ts:41]: Simple fix but also very easy to miss. Hence the unit tests.

File: tests/libs/__mocks__/prisma.ts [hunk 2]
```diff
@@ -7,6 +7,7 @@ import * as selects from "@calcom/prisma/selects";
 vi.mock("@calcom/prisma", () => ({
   default: prisma,
   prisma,
+  readonlyPrisma: prisma,
```
Comment (zomars) [inline tests/libs/__mocks__/prisma.ts:10]: This was causing vitest to complain when running unit tests.

**Review by zomars — COMMENTED** [review]
Ready for review

File: apps/api/pages/api/slots/_get.test.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+import dayjs from "@calcom/dayjs";
+
+import handler from "./_get";
+
+type CustomNextApiRequest = NextApiRequest & Request;
+type CustomNextApiResponse = NextApiResponse & Response;
+
+function buildMockData() {
+  prismock.user.create({
```
Comment (keithwillcode) [inline apps/api/pages/api/slots/_get.test.ts:16]: Is this a play on words or should it be prismaMock?

**Review by keithwillcode — APPROVED** [review]
Wow. Nice find. 

Love the addition of the unit tests.

File: apps/api/pages/api/slots/_get.test.ts (hunk 3 above)
Comment (zomars) [reply apps/api/pages/api/slots/_get.test.ts:16]: It is intended 

https://github.com/morintd/prismock

**Pull request merged**